packages/components/src/internal/components/samples/SampleStatusTag.tsx

```diff
@@ -22,6 +23,34 @@ interface Props {
     status: SampleStatus;
 }
 
+// exported for Jest test
+export function hexToRGB(hex: string): number[] {
+    const bigInt = parseInt(hex?.replace('#', ''), 16);
+    const r = (bigInt >> 16) & 255;
+    const g = (bigInt >> 8) & 255;
+    const b = bigInt & 255;
+    return [r, g, b];
+}
+
+// exported for Jest test
+export function getStatusTagStyle(status: SampleStatus): CSSProperties {
+    const color = getSampleStatusColor(status.color, status.statusType);
+    const style = SAMPLE_STATUS_COLORS[color];
+    if (style) {
+        return {
+            ...style,
+            borderColor: 'lightgray',
```

Comment:
I was just about to ask about this hardcoded value but then I saw the comment in for SAMPLE_STATUS_COLORS. Will that be decided before this is merged?

Reply:
Yes, should be decided with feature verification.